package/scripts/mmkv_headers.sh

```diff
@@ -0,0 +1,194 @@
+#!/bin/bash
+
+set -e
+
+# Script to set up MMKV headers for IDE support
+# This downloads or creates the necessary MMKV core headers without relying on the example app
```

Comment:
This can't be correct. I believe @lingol added prefab support to the MMKV gradle package, which means it should ship C++ headers. See [their build.gradle prefab spec](https://github.com/Tencent/MMKV/blob/c74d8b886abd87132c507ec17865a7fb4feb9679/Android/MMKV/mmkv/build.gradle#L111-L115).

So we definitely don't want to download them, and downloading is also not the same version as the one we have locally - so the symbols won't match.

Are you doing this just for IDE support, or will those headers _actually_ be included?